build2cmake/src/config/v2.rs

```diff
@@ -109,6 +109,7 @@ pub enum Kernel {
         cxx_flags: Option<Vec<String>>,
         depends: Vec<Dependencies>,
         rocm_archs: Option<Vec<String>>,
+        rocm_flags: Option<Vec<String>>,
```

Comment:
Maybe we should call them `hip_flags`? I think `cuda_flags` is a bit of a misnomer, in hindsight we should have called that `nvcc_flags`.